src/main/java/org/apache/commons/math4/analysis/differentiation/DSCompiler.java

```diff
@@ -1497,11 +1497,11 @@ public void cosh(final double[] operand, final int operandOffset,
 
         // create the function value and derivatives
         double[] function = new double[1 + order];
-        function[0] = FastMath.cosh(operand[operandOffset]);
+        final double function0 = function[0] = FastMath.cosh(operand[operandOffset]);
```

Comment:
`final` is useless, and not the code style for these files

Reply:
> final is useless

Yep, if you think it that way.

>and not the code style for these files

There is final for local variable for class `FieldDenseMatrix` , `TricubicInterpolator` , `UnivariatePeriodicInterpolator` , `FieldDenseMatrix` , `SobolSequenceGenerator`, `GeometryExample`, and several test classes.
So I think the `final` I used is not outside of "the code style for these files", thus no need to change.
Or if I mis-understand something, please tell me.
Thanks.